docs/src/pages/components/date-range-picker/CustomDateRangePickerDay.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
+import LocalizaitonProvider from '@material-ui/lab/LocalizationProvider';
+import DateRangePicker, { DateRange } from '@material-ui/lab/DateRangePicker';
+import DateRangePickerDay, {
+  DateRangePickerDayProps,
+} from '@material-ui/lab/DateRangePickerDay';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme) => ({
+  highlight: {
+    borderRadius: 0,
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.common.white,
+    '&:hover, &:focus': {
+      backgroundColor: theme.palette.primary.dark,
+    },
+  },
+  firstHighlight: {
+    borderTopLeftRadius: '50%',
+    borderBottomLeftRadius: '50%',
+  },
+  endHighlight: {
+    borderTopRightRadius: '50%',
+    borderBottomRightRadius: '50%',
+  },
+}));
+
+export default function CustomDateRangePickerDay() {
+  const classes = useStyles();
+  const [selectedDate, handleDateChange] = React.useState<DateRange<Date>>([
+    null,
+    null,
+  ]);
+
+  const renderWeekPickerDay = (
+    date: Date,
+    dateRangePickerDayProps: DateRangePickerDayProps<Date>,
+  ) => {
+    return (
+      <DateRangePickerDay
+        {...dateRangePickerDayProps}
+        className={clsx(dateRangePickerDayProps.className, {
+          [classes.firstHighlight]: dateRangePickerDayProps.isStartOfHighlighting,
+          [classes.endHighlight]: dateRangePickerDayProps.isEndOfHighlighting,
+          [classes.highlight]: dateRangePickerDayProps.isHighlighting,
+        })}
+      />
+    );
+  };
+
+  return (
+    <LocalizaitonProvider dateAdapter={AdapterDateFns}>
+      <DateRangePicker
+        label="Week picker"
```

Comment:
Are you sure about this label? It doesn't seem to be a week picker.
```suggestion
        label="Customized day"
```

Reply:
c&p